packages/apps/job-launcher/server/src/modules/payment/payment.dto.ts

```diff
@@ -43,18 +42,16 @@ export class PaymentCryptoCreateDto {
 export class PaymentCreateDto {
   public transactionHash?: string;
   public amount?: string;
-  public currency?: Currency;
+  public currency?: string;
```

Comment:
Why did you remove the Currency type?

Reply:
It was incorrect a bit, we can store payment in fiat and in crypto, that means currency represent as fiat currencies and tokens symbols at the same time. I don't actually want to use double types for property, for example `currency: Currency | TokenId`, at the same time not good idea to union Currency and TokenId to the same enum type. Probably it can be improved somehow.